File: Python/Fibonnaci.py
n = int(input("Enter the nth number: "))
a, b = 0, 1

def fib(n, a, b):
    
    if n == 1:
        print(0)
    elif n > 1:
        print(0, 1, end = " ")
        for i in range(n-2):
            j = a+b
            print(j,end = " ")
            a = b
            b = j    
            i += 1
    else:
        print("Invalid Input")

fib(n, a, b)

File: C/Fibonnaci.c
#include <stdio.h>
#include <stdlib.h>
#include <math.h>

int fib(int n, int a, int b){

    if(n==1){ printf("%d", 0); }
    else if(n > 1){
        printf("%d %d ", 0, 1);
        for(int i = 0; i < n-2; i++){
            int j = a + b;
            printf("%d ", j);
            a = b;
            b = j;
        }
    } else {
        printf("Invalid Input");
    }

    return 0;
}

int main(){

    int n;
    int a = 0;
    int b = 1;

    scanf("%d", &n);

    fib(n, a, b);


    return 0;
}
